devstack/lib/nfp

```diff
@@ -1,18 +1,26 @@
 #!/bin/bash
 
 NFPSERVICE_DIR=$DEST/gbp
-DISK_IMAGE_DIR=$DEST/gbp/gbpservice/tests/contrib
 TOP_DIR=$PWD
+DISK_IMAGE_DIR={DISK_IMAGE_DIR:-$DEST/gbp/gbpservice/tests/contrib}
 
 NEUTRON_CONF_DIR=/etc/neutron
 NEUTRON_CONF=$NEUTRON_CONF_DIR/neutron.conf
 
 NFP_CONF_DIR=/etc/nfp
 echo "TOP-DIR-NFP : $PWD"
 
+function create_nfp_image {
+    TOP_DIR=$TOP_DIR
+    sudo python $DISK_IMAGE_DIR/diskimage-create/disk_image_create.py $DISK_IMAGE_DIR/diskimage-create/conf.json
+    BUILT_IMAGE_PATH=$(cat /tmp/nfp_image_path)
+    upload_image file://$BUILT_IMAGE_PATH
+
+    openstack --os-cloud=devstack-admin flavor create --ram 512 --disk 3 --vcpus 1 m1.nfp-tiny
+}
+
 function prepare_nfp_image_builder {
-    #setup_develop $NFPSERVICE_DIR
-    sudo -H -E pip install -r $DISK_IMAGE_DIR/diskimage-create/requirements.txt
+    sudo -H -E pip install -r $NFPSERVICE_DIR/diskimage-create/requirements.txt
```

Comment:
path for diskimage-create/requirements.txt is wrong. Please check.
It should be $DISK_IMAGE_DIR/diskimage-create/requirements.txt